### DISCUSSION TITLE
월드 파일 등록이 되지 않습니다.

### DISCUSSION THREAD

**Discussion by ussam78** [naverz/zepeto-studio-kor#901] [5. Q&A]
2일 동안 계속 월드 빌드 중 상태인데 어떻게 해야 하나요? 

<img width="1146" alt="스크린샷 2022-06-20 오후 2 48 14" src="https://user-images.githubusercontent.com/36125434/174533745-58f9e032-740d-4881-8bfc-b42426464f29.png">

**Comment by Zepeto-TechEvangelist** [comment]
안녕하세요, https://github.com/naverz/zepeto-studio-kor/issues/886 과 같은 원인을 수 있어서 내부애서 확인 중입니다.

**Comment by Zepeto-TechEvangelist** [comment]
안녕하세요, 내부에서 확인 결과 패키지 무한로딩 이슈는 스크립트가 많고 순환참조가 많은 케이스에서 발생하고 있습니다.
이를 부분을 개선하는 제페토 스크립트 패키지의 preview버전을 배포하였습니다.

Window → Package Manager 버튼을 눌러서 Package Manager를 오픈 →  [ZEPETO.Script]를 누르고 오른쪽 하단의 Update 버튼을 누릅니다.
1.3.4 preview 버전 업 후 컴파일러 재실행을 위해 유니티를 재시작 해 주세요.

<img width="797" alt="스크린샷 2022-06-21 오후 8 03 39" src="https://user-images.githubusercontent.com/100118650/174784909-cdeea83d-0881-4f56-a6cf-7a58a9a4415c.png">

위 방법으로 우선 적용 부탁 드립니다.

감사합니다.
